common/repositories/base/base_spells_new_repository.h

```diff
@@ -199,8 +199,8 @@ class BaseSpellsNewRepository {
 		int         pvpresistcalc;
 		int         pvpresistcap;
 		int         spell_category;
-		int         field181;
```

Comment:
Did you manually update these repositories or did you use the script to update ?

Reply:
The script is unusable for me so I manually updated it.

Reply:
What do you mean, unusable?

Reply:
How to use is here: https://eqemu.gitbook.io/server/in-development/developer-area/repositories

```
perl ~/code/utils/scripts/generators/repository-generator.pl ~/server spells_new
```
Checked out your code and looks like you got it all manually 100% anyways